composer.json

```diff
@@ -18,6 +18,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "~4.8 || ~5.0",
+        "doctrine/data-fixtures": "^1.1",
         "symfony/monolog-bundle": "~2.4",
         "jackalope/jackalope-doctrine-dbal": "~1.2",
         "zendframework/zendsearch": "2.*@RC"
```

Comment:
would be great if you could also require the `php-cs-fixer` dependency and execute it in the ci workflow and remove PrettyCI completely

Reply:
Didn't we add the php-cs-fixer only for `> 2.0` versions?

Reply:
i think thats not consistent between different bundles. for me its okay to only add it in the `2.x` branch 🙂

Reply:
yes only for `2.x` version we can add the php-cs-fixer as a dependencies, as the requireing it would increase the prefer lowest when install with dev dependencies and so prefer lowest is not longer correctly tested.

Reply:
that's also fine for me 👍

Reply:
If we do not add `php-cs-fixer` here, we should at least fix the current errors reported by `pretty`.

Reply:
Removing `pretty` is fine too, therefore I'll resolve this